Zotero/Assets/en.lproj/Localizable.strings

```diff
@@ -221,6 +221,7 @@
 "pdf.search.one_match" = "Found 1 match";
 "pdf.search.multiple_matches" = "Found %d matches";
 "pdf.annotation_popover.title" = "Edit Annotation";
+"pdf.annotation_popover.share" = "Share Annotation";
```

Comment:
I think we'll want this to be "Share Image". Sharing an entire annotation implies sharing the comment and citation, as we do for annotation Quick Copy on the desktop, and we'll support that on iOS later (though doing it for image annotations is complicated).